libs/dkg/src/dkg.cpp

```diff
@@ -754,7 +719,53 @@ bool DistributedKeyGeneration::BuildQual()
 }
 
 /**
- * If in qual a member Computes individual share of the secret key and further computes and
+ * Checks coefficients sent by qual members and puts their address and the secret shares we received
+ * from them into a complaints maps if the coefficients are not valid
+ *
+ * @return Map of address and pair of secret shares for each qual member we wish to complain against
+ */
+std::unordered_map<
```

Comment:
Stylistic: having a ```using``` for these types would avoid it getting forced onto multiple lines. Also, does iq mean anything in this context?

Reply:
I have replaces the unordered_map with an alias. iq should not be there and has been removed in the tests branch I am working on. I will remove it here as well.

Reply:
I have replaces the unordered_map with an alias.